### ISSUE TITLE
Use portable CI scripts

### ISSUE THREAD

**Issue by Arthelon** [SkygearIO/auth-landing#69]

**Comment by Arthelon** [comment]
@kiootic For this, did you mean to rely on custom CI scripts as much as possible and not use external actions?

**Comment by kiootic** [comment]
Yes, for example, after the checkout action, the only script to run should be something like `npm run ci`
